build/tekton/pipeline/pipeline.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: n3wscott-sources-pipeline
+spec:
+  resources:
+    - name: source-repo
+      type: git
+  tasks:
+    - name: lint
+      taskRef:
+        name: golangci-lint
+      params:
+        - name: package
+          value: ${inputs.resources.source-repo.package}
```

Comment:
This is not supported (yet ?) as a resource can't have custom parameters name.
Not that there is some movement to rework the PipelineResource model so that something like that could be possible (in a different way(